rust/openvasctl/src/pref_handler.rs

```diff
@@ -76,7 +76,7 @@ where
         Ok(())
     }
 
-    fn prepare_scan_id_for_openvas(&mut self) -> RedisStorageResult<()> {
+    async fn prepare_scan_id_for_openvas(&mut self) -> RedisStorageResult<()> {
         self.redis_connector.push_kb_item(
```

Comment:
`push_kb_item` doesn't take so long. Its quite fast, since an item can be a vector of items. It would be different if it has to store items one by one and do many calls to the push function. In any case, we can check this later and do it given the case.